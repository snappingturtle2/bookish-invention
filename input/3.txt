1
HEX_STRING
1111
DUMP
MUL_UINT64
0 0 16
DUMP
END
