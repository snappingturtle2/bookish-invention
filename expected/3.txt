0x1111

0x011110

